contracts/supplementary/rebalancing-manager/BTCETHRebalancingManager.sol

```diff
@@ -58,18 +56,17 @@ contract BTCETHRebalancingManager {
     address public setTokenFactory;
 
     address public btcPriceFeed;
-    IMedian private btcPriceFeedInstance;
-
     address public ethPriceFeed;
-    IMedian private ethPriceFeedInstance;
 
-    address public core;
-    ICore private coreInstance;
+    address public coreAddress;
 
-    uint256 public btcMultiplier;
-    uint256 public ethMultiplier;
     address public auctionLibrary;
     uint256 public auctionTimeToPivot;
+    uint256 public btcMultiplier;
+    uint256 public ethMultiplier;
+
+    uint256 public lowerAllocationBound;
+    uint256 public upperAllocationBound;
```

Comment:
Same comment as other PR: Can we call this `minimumUpperThreshold` and `maximumLowerThreshold` to match rebalance strategy terminology?